internal_testsuites/golang/testsuite/startosis_add_service_test/startosis_add_service_testsuite_test.go

```diff
@@ -0,0 +1,47 @@
+package startosis_add_service_test
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis-cli/golang_internal_testsuite/test_helpers"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/enclaves"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+const (
+	name                  = "startosis-add-service"
+	isPartitioningEnabled = false
+)
+
+type StartosisAddServiceTestSuite struct {
```

Comment:
That's super cool the concept of suite here, I think we could probably do something similar with the SL framework test we have